keras/preprocessing/sequence.py

```diff
@@ -216,3 +218,93 @@ def _remove_long_seq(maxlen, seq, label):
             new_seq.append(x)
             new_label.append(y)
     return new_seq, new_label
+
+
+class TimeSeriesSequence(Sequence):
```

Comment:
Not sure about this name. `TimeSeriesSequence` is redundant. It won't immediately be clear to users that "sequence" in the name refers to the Keras `Sequence` class.

Possibilities:

`TimeseriesGenerator`
`TimeseriesSampler`

Other ideas?

Reply:
`TimeseriesGenerator` seems the most appropriate to me, and makes the use-case clearer too. Renaming it to that.